Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -458,8 +463,11 @@ def extract_from_xml(xml, path):
     return xml
 
 
-def generate_field_contents(fields_values, level_fields):
-    fields_values = json.loads(fields_values)
+def generate_field_contents(client, fields_values, level_fields):
+    try:
+        fields_values = json.loads(fields_values)
```

Comment:
is it possible that it will already be a dictionary?
for example, when running it from a script or when using context as the argument